Microsoft.Recognizers.Text.Number/BaseNumberRecognizer.cs

```diff
@@ -1,27 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace Microsoft.Recognizers.Text.Number
-{
-    public abstract class BaseNumberRecognizer : Recognizer<Dictionary<string, Dictionary<Type, IModel>>>
-    {
-        public static IModel GetModel<TModel>(string culture, bool fallbackToDefaultCulture = true)
-        {
-            if (!ModelInstances.ContainsKey(culture) || !ModelInstances[culture].ContainsKey(typeof(TModel)))
-            {
-                if (fallbackToDefaultCulture)
-                {
-                    culture = DefaultCulture;
-                }
-                else
-                {
-                    throw new Exception($"ERROR: No IModel instance for {typeof(TModel)}");
-                }
-            }
-
-            var model = ModelInstances[culture][typeof(TModel)];
-
-            return model;
-        }
-    }
-}
+using System;
```

Comment:
I was working with @mattshim for some reason his project did not have the text project.  The part that actually changed is in Microsoft.Recognizers.Text.Number/NumberRecognizer.cs Added the French to it.  For some reason it is not getting captured in the pull request, I'll close this one and resubmit